src/redprl/metalanguage/evaluate.fun

```diff
@@ -134,7 +134,7 @@ struct
 
          val check =
            if subgoalsCount = 0 then () else
-             RedPrlLog.print RedPrlLog.WARN (pos, Fpp.hsep [Fpp.text @@ Int.toString subgoalsCount, Fpp.text "Remaining Obligations"])
+             RedPrlLog.print RedPrlLog.WARN (pos, Fpp.hsep [Fpp.text @@ Int.toString subgoalsCount, Fpp.text @@ "Remaining Obligation" ^ (if subgoalsCount = 1 then "" else "s")])
```

Comment:
I suggest using the whole words "Obligation" and "Obligations" so that future localization would be easier.